casbin/util/util.py

```diff
@@ -57,3 +57,30 @@ def set_subtract(a, b):
             diff.append(x)
 
     return diff
+
+def has_eval(s):
+    '''determine whether matcher contains function eval'''
+    line = s.split(" && ")
```

Comment:
I'm sure that the implement is incorrect.

consider the following scenario:
```
eval(p.sub_rule) || eval(p.obj_rule)
eval(p.sub_rule) || p.obj == r.obj && eval(p.domain_rule)
```

I recommend use regex to implement the feature.

Reply:
I've fixed the error by using reg expression to implement that.